lib/landrush/cap/all/read_host_visible_ip_address.rb

```diff
@@ -0,0 +1,49 @@
+module Landrush
+  module Cap
+    module All
+      module ReadHostVisibleIpAddress
+        def self.filter_addresses(addresses)
+          unless @machine.config.landrush.exclude.nil?
+            re = Regexp.union(@machine.config.landrush.exclude)
+
+            addresses = addresses.select do |addr|
+              !addr['name'].match(re)
+            end
+          end
+
+          addresses
+        end
+
+        def self.read_host_visible_ip_address(machine)
+          @machine = machine
+
+          addr      = nil
+          addresses = machine.guest.capability(:landrush_ip_get)
```

Comment:
Yep. And because the cap will be supported on all platforms (at least, I don't think Vagrant has a guest that Go doesn't support), this code will apply to all platforms.